lua/nvim-treesitter-refactor/navigation.lua

```diff
@@ -33,24 +33,77 @@ function M.list_definitions(bufnr)
 
   local qf_list = {}
 
-  for _, def in ipairs(definitions) do
-    locals.recurse_local_nodes(def, function(_, node, _, match)
-      local lnum, col, _ = node:start()
-
-      table.insert(qf_list, {
-        bufnr = bufnr,
-        lnum = lnum + 1,
-        col = col + 1,
-        text = ts_utils.get_node_text(node)[1] or "",
-        kind = match and match:sub(1, 1) or ""
-      })
-    end)
+  for _, node in ipairs(definitions) do
+    local lnum, col, _ = node.node:start()
```

Comment:
You'll have to use the recurse_local_nodes function here because there's no guarantee that the node will only be one deep.

Reply:
That's done in https://github.com/nvim-treesitter/nvim-treesitter/pull/491